django_hashedfilenamestorage/tests.py

```diff
@@ -130,7 +130,7 @@ def patch(namespace, **values):
         if namespace._wrapped is None:
             namespace._setup()
         namespace = namespace._wrapped
-    for (name, value) in values.iteritems():
+    for (name, value) in list(values.items()):
```

Comment:
Does this need to be cast to a list?

Reply:
No, that's 2to3 being over-cautious again; will fix